apps/mobile/src/kernel/i18n/useStrings.ts

```diff
@@ -1793,6 +1793,32 @@ export const useStrings = () => {
         // bluetoothEnabled: f(setupWalletMessages.bluetoothEnabled),
         // locationEnabled: f(setupWalletMessages.locationEnabled),
       },
+
+      // Mapped strings for dynamic category mapping
+      useMappedStrings: () => {
```

Comment:
maybe rename to getCDapCategory strings, all React extensions think that this is hook :D 
```suggestion
      useMappedStrings: () => {
```